client/state-trie-migration-rpc/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "sc-state-trie-migration-rpc"
```

Comment:
This crate should also be moved under the pallet. We already have multiple of these pallet rpc crates.

Reply:
I had it under pallet at first but the dependencies from client crate sc-service to pallet was rejected.
Wait, I don't have this dependency anymore, think I can move all under pallet now (even move primitive to rpc as suggested) :+1:

Reply:
Oh, actually rpc requires dependency to clients crate, which is not allowed (other frame crate can do without because they are using the RuntimeApi interface).

Reply:
Then please move it to the `utils/frame/rpc` folder

Reply:
> Then please move it to the `utils/frame/rpc` folder

That's cheating, but yes it results in one less crate, I'm happy with it.